balancer-js/package.json

```diff
@@ -20,18 +20,19 @@
     "dist/"
```

Comment:
here I added "address-books:generate" script and sorted the scripts alphabetically